src/components/infiniteScroll/index.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import * as React from 'react';
+import PullToRefresh from 'pulltorefreshjs';
```

Comment:
@mxstbr any particular reason, why we need to implement only in `InfiniteScroll` component ??